library/ssl_tls13_server.c

```diff
@@ -22,18 +22,203 @@
 #if defined(MBEDTLS_SSL_SRV_C) && defined(MBEDTLS_SSL_PROTO_TLS1_3)
 
 #include "mbedtls/debug.h"
+#include "mbedtls/error.h"
+#include "mbedtls/platform.h"
+#include <string.h>
```

Comment:
Is string.h used in any of the macros such as `MBEDTLS_SSL_DEBUG_MSG`? I may have missed it, but I could not spot a member function of it being invoked in the introduced code.